packages/dashboard/src/components/cardGrid/index.js

```diff
@@ -51,7 +51,7 @@ DashboardGrid.propTypes = {
 };
 
 const CardGrid = forwardRef(function CardGrid(
-  { ariaLabel, children, pageSize, isPosterHeight },
```

Comment:
💭 If we're not using this anymore then it should be removed from the following files too:
- `packages/dashboard/src/app/views/myStories/content/storyGridView/index.js`
- `packages/dashboard/src/app/views/exploreTemplates/content/templateGridView.js`

Reply:
thanks, overlooked those!